.github/workflows/build.yml

```diff
@@ -70,33 +74,25 @@ jobs:
       - name: Install necessary dependencies
         run: |
           pip install pyyaml
-          pip install git+https://github.com/eduNEXT/tvm.git
 
       - name: Get Tutor Configurations from config.yml and set them as an environment variable
+        working-directory: ${{ github.workspace }}
         env:
-          REQUIRED_KEYS: TUTOR_VERSION TUTOR_APP_NAME
+          REQUIRED_KEYS: TUTOR_VERSION
           OPTIONAL_KEYS: DOCKER_REGISTRY
           CONFIG_FILE: strains/${{ inputs.STRAIN_PATH }}/config.yml
           SCRIPT_PATH: picasso/.github/workflows/scripts/get_tutor_config.py
         run: |
           ENV_VARS=$(python $SCRIPT_PATH --config-file $CONFIG_FILE --required-keys $REQUIRED_KEYS --optional-keys $OPTIONAL_KEYS)
           echo "$ENV_VARS" >> $GITHUB_ENV
 
-      - name: Configure TVM project
-        working-directory: strains/${{ inputs.STRAIN_PATH }}
+      - name: Install Tutor and its dependencies from the version specified in the config.yml
         run: |
-          tvm install $TUTOR_VERSION
-          tvm project init $TUTOR_APP_NAME $TUTOR_VERSION
-
-          # This command copies all the files and folders
-          # from the repository STRAIN_PATH to the recently above created TVM PROJECT folder
-          cp -r $(ls --ignore=$TUTOR_APP_NAME) $TUTOR_APP_NAME/
+          pip install tutor[full]==$TUTOR_VERSION
```

Comment:
This is not a blocker comment, but, taking advantage of the fact we are no longer using tvm, why not make a pip install in the form of git, to allow nightly builds?

Example: pip install git+git@github.com:overhangio/tutor.git@$TUTOR_VERSION

I'm not sure what is the demand for nightly builds, but if we have some, we can use the git version. If not, leave it as it is because it is more explicit.

Reply:
I'll do some testing to see if we can do the same there. Thanks for the suggestion!

Reply:
I forgot to say, but I had already applied the suggestion: https://github.com/eduNEXT/picasso/pull/25/commits/9176b114f600071abd1b84dba1bad5b2a5c2c9fe. Thanks again!